charts/pulsar/values.yaml

```diff
@@ -530,18 +530,19 @@ zookeeper:
       failureThreshold: 10
       initialDelaySeconds: 10
       periodSeconds: 30
-      timeoutSeconds: 5
+      timeoutSeconds: 10
```

Comment:
Why do we increase timeout from 5 to 10?

Reply:
If no timeout set or with short time timeout, sometimes will get zk error like. Not sure about the RCA for this. 

```
12:07:55.432 [QuorumPeer[myid=1](plain=0.0.0.0:2181)(secure=disabled)] INFO  org.apache.zookeeper.server.quorum.FastLeaderElection - Notification time out: 51200
12:07:55.433 [QuorumConnectionThread-[myid=1]-1] WARN  org.apache.zookeeper.server.quorum.QuorumCnxManager - Cannot open channel to 3 at election address sn-pulsar-zookeeper-2.sn-pulsar-zookeeper.sn.svc.cluster.local:3888
java.net.UnknownHostException: sn-pulsar-zookeeper-2.sn-pulsar-zookeeper.sn.svc.cluster.local
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:229) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:?]
	at java.net.Socket.connect(Socket.java:609) ~[?:?]
	at org.apache.zookeeper.server.quorum.QuorumCnxManager.initiateConnection(QuorumCnxManager.java:383) ~[io.streamnative-zookeeper-3.6.3.1.jar:3.6.3.1]
	at org.apache.zookeeper.server.quorum.QuorumCnxManager$QuorumConnectionReqThread.run(QuorumCnxManager.java:457) ~[io.streamnative-zookeeper-3.6.3.1.jar:3.6.3.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
```